server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -152,4 +137,61 @@ private String getVideoId(String response) {
     // Only returning the first for now, potential to change if I optimize search results.
     return searchResults.get(0);
   }
+
+  /**
+   * Parses the comment JSON string and retrieves the comments.
+   *
+   * @param response The JSON string to be parsed.
+   * @return An ArrayList with each comment
+   */
+  private ArrayList<String> retrieveComments(String response) {
+    ArrayList<String> commentList = new ArrayList<>();
+
+    // Traversing to the items array
```

Comment:
Some of these comments aren't necessary because they just describe the mechanics of the code (that an array of items is being created) rather than the otherwise clear intent of the code.